web_src/js/features/common-global.js

```diff
@@ -186,6 +186,68 @@ export function initGlobalDropzone() {
   }
 }
 
+function linkAction(e) {
+  e.preventDefault();
+
+  // A "link-action" can post AJAX request to its "data-url"
+  // Then the browser is redirect to: the "redirect" in response, or "data-redirect" attribute, or current URL by reloading.
+  // If the "link-action" has "data-modal-confirm(-html)" attribute, a confirm modal dialog will be shown before taking action.
```

Comment:
`link-action` is a bit unclear to me. Why not call it `fetch-action` and maybe also use `fetch` API?

Reply:
There are a lot. So, just re-use the existing name, and refactor later.